Source/WebCore/platform/MediaStrategy.cpp

```diff
@@ -37,4 +42,28 @@ std::unique_ptr<NowPlayingManager> MediaStrategy::createNowPlayingManager() cons
     return makeUnique<NowPlayingManager>();
 }
 
+#if ENABLE(MEDIA_SOURCE)
+void MediaStrategy::setMockMediaSourceEnabled()
+{
+#if USE(AVFOUNDATION)
+    WebCore::DeprecatedGlobalSettings::setAVFoundationEnabled(false);
+#endif
+#if USE(GSTREAMER)
+    WebCore::DeprecatedGlobalSettings::setGStreamerEnabled(false);
+#endif
+
+    enableMockMediaSource();
+}
+
+bool MediaStrategy::mockMediaSourceEnabledInRemoteEngine() const
+{
+    return false;
+};
+
+void MediaStrategy::enableMockMediaSource()
```

Comment:
If we update setMockMediaSourceEnabled to enableMockMediaSource, we would need to rename this one to something like addMockMediaSourceEngine

Reply:
ok